src/androidTest/java/com/couchbase/lite/performance/Test3_CreateDocsWithAttachments.java

```diff
@@ -80,8 +73,23 @@ public boolean run() {
                 return true;
             }
         });
+        assertTrue(success);
 
-        Log.v("PerformanceStats", TAG + "," + Long.valueOf(System.currentTimeMillis() - startMillis).toString() + "," + getNumberOfDocuments() + "," + getSizeOfAttachment());
+        Log.e("PerformanceStats", TAG + "," + Long.valueOf(System.currentTimeMillis() - startMillis).toString() + "," + getNumberOfDocuments() + "," + getSizeOfAttachment());
+    }
+
+    private static Map<String, Map<String, Object>> getAttachmentsDict(byte[] data, String name, String type, boolean gzipped) {
+        if (gzipped)
+            // TODO
+            ;
+        Map<String, Object> att = new HashMap<String, Object>();
+        att.put("content_type", type);
+        att.put("data", data);
+        if (gzipped)
+            att.put("encoding", "gzip");
```

Comment:
There is a TODO comment for gzipped above but not here? Could you please clarify?